core/trino-main/src/main/java/io/trino/execution/scheduler/PipelinedQueryScheduler.java

```diff
@@ -220,13 +223,18 @@ public PipelinedQueryScheduler(
         this.dynamicFilterService = requireNonNull(dynamicFilterService, "dynamicFilterService is null");
         this.tableExecuteContextManager = requireNonNull(tableExecuteContextManager, "tableExecuteContextManager is null");
         this.splitSourceFactory = requireNonNull(splitSourceFactory, "splitSourceFactory is null");
+        this.schedulerSpan = tracer.spanBuilder("scheduler")
```

Comment:
looks you are not closing this one. Should happen probably in listener on `queryStateMachine`

Reply:
Done